### PR TITLE
LRQA-69013

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by leticiachiu** [comment]
ci:test:sf

**Comment by leticiachiu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase [hunk 1]
```diff
@@ -18,6 +18,7 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
+
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsGridField.testcase:21]: @leticiachiu, is this really necessary?

**Comment by leticiachiu** [comment]
@natocesarrego fixed

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**